Makefile

```diff
@@ -0,0 +1,35 @@
+DATE		= $(shell date)
+PYTHON		= $(shell which python)
+NOSETESTS = $(shell which nosetests)
```

Comment:
Also because:

```
Nose has been in maintenance mode for the past several years and will likely cease without a new person/team to take over maintainership. New projects should consider using Nose2, py.test, or just plain unittest/unittest2.
```

See: https://nose.readthedocs.io/en/latest/#note-to-users